billboard.java

```diff
@@ -0,0 +1,51 @@
+
+/*
+Bill Zheng
+ */

```

Comment:
Need:
* time you spent
* test results
* difficulty [1-10]
* feedback/reflection